ipn/ipnserver/server.go

```diff
@@ -747,6 +748,9 @@ func Run(ctx context.Context, logf logger.Logf, ln net.Listener, store ipn.State
 	if err != nil {
 		return err
 	}
+	if ns != nil {
+		ns.SetLocalBackend(server.LocalBackend())
+	}
```

Comment:
this is the missing wiring that `tailscaled.go` did but I forgot to do in `tailscaled_windows.go`.

Reply:
can ns ever be nil here?

Reply:
In the test only I think? I just didn't want to think about it and deal with crashes later.